tests/test_types.py

```diff
@@ -1866,6 +1867,17 @@ class Model(BaseModel):
     assert m.size.human_readable(decimal=True) == human_dec
 
 
+def test_bytesize_to():
+    class Model(BaseModel):
+        size: ByteSize
+
+    m = Model(size='1GiB')
+
+    assert pytest.approx(m.size.to('MiB')) == 1024
```

Comment:
```suggestion
    assert m.size.to('MiB') == pytest.approx(1024)
```

reads better and is what's suggested by [the docs](https://docs.pytest.org/en/latest/reference.html#pytest-approx)